nixos/modules/services/hardware/actkbd.nix

```diff
@@ -95,6 +111,51 @@ in
         '';
       };
 
+      volumeControl = {
+
+        enable = mkOption {
+          type = types.bool;
+          default = false;
+          description = ''
+            Whether to enable ALSA volume and capture control with keyboard media keys.
+
+            You want to leave this disabled if you run a desktop environment
+            like KDE, Gnome, Xfce, etc, as those handle such things themselves.
+            You might want to enable this if you run a minimalistic desktop
+            environment or work from bare linux ttys/framebuffers.
+          '';
+        };
+
+        step = mkOption {
+          type = types.string;
+          default = "1";
+          example = "1%";
+          description = ''
+            The value by which to increment/decrement volume on media keys.
+
+            See amixer(1) for allowed values.
+          '';
+        };
+
+      };
+
+      brightnessControl = {
+        enable = mkEnableOption ''
+          Whether to enable display brightness control via sysfs facility.
+          Step percentage is adjusted in accordance with hardware maximum
+          brightness value.
+        '';
```

Comment:
`mkEnableOption` already uses `description = "Whether to enable ${name}.";` (see https://github.com/NixOS/nixpkgs/blob/60e8fcf0e56ce881294b06bd1f40069016308162/lib/options.nix).